float32 vxBody
float32 vyBody
float32 vzBody
float32 yawRate
